tools/generators/move-packages/index.ts

```diff
@@ -154,3 +163,17 @@ function updateCodeOwners(tree: Tree, schema: MovePackagesGeneratorSchema) {
     tree.write(codeownersPath, updatedCodeOwnersFile);
   }
 }
+
+function updateApiExtractor(tree: Tree, schema: MovePackagesGeneratorSchema) {
+  const projectConfig = getProjectConfig(tree, { packageName: schema.name! });
+  const apiExtractorLocalPath = joinPathFragments(projectConfig.paths.configRoot, 'api-extractor.local.json');
+  const apiExtractorLocal = readJson(tree, apiExtractorLocalPath);
+  const originalEntryPointPath = apiExtractorLocal.mainEntryPointFilePath;
+  const updatedEntryPointPath = originalEntryPointPath.replace(
+    'packages/<unscopedPackageName>',
+    `packages/${schema.destination}/<unscopedPackageName>`,
```

Comment:
`schema.destination` in the case of v9 moves will be along the lines of `testFolder/react-menu` so adding the whole `destination` will be redundant with `unscopedPackageName`

Reply:
hmm, right that was bit unclear. so what should we do ? strip the last dirname part from destination ?

Reply:
Yup that should do it since the last part of the destination will always be the package name anyway

Reply:
should be fixed c054783087882b175568f518d668948b4e2d3663